BUILD.md

```diff
@@ -1,4 +1,10 @@
-# Building ZFSBootMenu Quick Start Guide
+# Building ZFSBootMenu with ZBM-Builder - A Quick Start Guide
+
+## Introduction
+
+The goal of `zbm-builder.sh` is to provide a simple, straightforward and relatively agnostic method of building locally configured ZFSBootMenu images without the additional complexities of either driving container management tools and build containers directly, or installing and integrating ZFSBootMenu within an existing operating system hierarchy.
+
+While the run-time components comprising ZFSBootMenu (i.e. the `kernel`, matching `initramfs` and/or integrated `EFI` bundle) can be built either inside or outside of a containerised envirnonment, using the pre-built container is preferred - it is considered the sanctioned (and, not to mention, most *tested*) method for doing so. Therefore, this document serves as a guide for building ZFSBootMenu using this technique.
```

Comment:
Strip this whole sentence. It's not really accurate. Container builds exist for distributions that don't package ZFSBootMenu (everybody but Void). It is *a* way to build it, but not the preferred way.

Reply:
Will do :+1: